.github/workflows/elasticsearch.yml

```diff
@@ -6,6 +6,9 @@ jobs:
   run-action:
     name: Start Elasticsearch
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        elasticsearch: ["5.6.16", "6.8.9", "7.7.0", "8.0.0-SNAPSHOT"]
```

Comment:
Can you please use the following versions?
```
6.8-SNAPSHOT
7.x-SNAPSHOT
8.0-SNAPSHOT
```

Elasticsearch 5 is no longer supported :)

Reply:
Thank you your comment.
I'll fix it.
please wait a few minutes.